File: consulting/src/main/kotlin/it/unibo/lss/fcla/consulting/application/persistence/FreelancerRepository.java
package it.unibo.lss.fcla.consulting.application.persistence;

import it.unibo.lss.fcla.consulting.common.EventSourcedRepository;
import it.unibo.lss.fcla.consulting.domain.freelancer.Freelancer;

/**
 * @author Stefano Braggion
 */
public class FreelancerRepository extends EventSourcedRepository<Freelancer> {
    public FreelancerRepository(EventStore eventStore) {
        super(eventStore);
    }
}

File: consulting/src/main/kotlin/it/unibo/lss/fcla/consulting/application/persistence/FreelancerRepository.kt
package it.unibo.lss.fcla.consulting.application.persistence

import it.unibo.lss.fcla.consulting.common.EventSourcedRepository
import it.unibo.lss.fcla.consulting.domain.freelancer.Freelancer

/**
 * @author Stefano Braggion
 *
 * Concrete implementation of a [EventSourcedRepository] for [Freelancer] aggregate
 */
class FreelancerRepository(eventStore: EventStore) : EventSourcedRepository<Freelancer>(eventStore)
